util/topgen/templates/chiplevel.sv.tpl

```diff
@@ -0,0 +1,1197 @@
+    .clk_main_i                   ( clk_main         ),
+    .clk_io_i                     ( clk_main         ),
+    .clk_usb_i                    ( clk_usb_48mhz    ),
+    .clk_aon_i                    ( clk_aon          ),
+    .clks_ast_o                   (                  ),
+    .clk_main_jitter_en_o         (                  ),
+    .rsts_ast_o                   (                  ),
+    .pwrmgr_ast_req_o             (                  ),
+    .pwrmgr_ast_rsp_i             ( ast_base_pwr     ),
+    .sensor_ctrl_ast_alert_req_i  ( ast_base_alerts  ),
+    .sensor_ctrl_ast_alert_rsp_o  (                  ),
+    .sensor_ctrl_ast_status_i     ( ast_base_status  ),
+    .usbdev_usb_ref_val_o         (                  ),
+    .usbdev_usb_ref_pulse_o       (                  ),
+    .ast_edn_req_i                ( '0               ),
+    .ast_edn_rsp_o                (                  ),
+    .flash_bist_enable_i          ( lc_ctrl_pkg::Off ),
+    .flash_power_down_h_i         ( 1'b0             ),
+    .flash_power_ready_h_i        ( 1'b1             ),
+    .ast_clk_byp_req_o            ( lc_clk_bypass    ),
+    .ast_clk_byp_ack_i            ( lc_clk_bypass    ),
+
+% if target["name"] != "cw305":
+    .ast_tl_req_o                 (                  ),
+    .ast_tl_rsp_i                 ( '0               ),
+    .otp_ctrl_otp_ast_pwr_seq_o   (                  ),
+    .otp_ctrl_otp_ast_pwr_seq_h_i ( '0               ),
+    .es_rng_req_o                 (                  ),
+    .es_rng_rsp_i                 ( '0               ),
+    .es_rng_fips_o                (                  ),
+    .pinmux2ast_o                 (                  ),
+    .ast2pinmux_i                 ( '0               ),
+% endif
+
+    // Multiplexed I/O
+    .mio_in_i        ( mio_in   ),
+% if target["name"] == "cw305":
+    .mio_out_o       ( mio_out_pre  ),
+% else:
+    .mio_out_o       ( mio_out  ),
+% endif
+    .mio_oe_o        ( mio_oe   ),
+
+    // Dedicated I/O
+    .dio_in_i        ( dio_in   ),
+    .dio_out_o       ( dio_out  ),
+    .dio_oe_o        ( dio_oe   ),
+
+    // Pad attributes
+    .mio_attr_o      ( mio_attr      ),
+    .dio_attr_o      ( dio_attr      ),
+
+    // Memory attributes
+    .ram_1p_cfg_i    ( '0 ),
+    .ram_2p_cfg_i    ( '0 ),
+    .rom_cfg_i       ( '0 ),
+
+    // DFT signals
+    .scan_rst_ni     ( 1'b1          ),
+    .scan_en_i       ( 1'b0          ),
+    .scanmode_i      ( lc_ctrl_pkg::Off )
+  );
+% endif
+
+
+###################################################################
+## CW305 capture trigger                                         ##
+###################################################################
+% if target["name"] == "cw305":
+
+  //////////////////////////////////////
+  // Generate precise capture trigger //
+  //////////////////////////////////////
+
+  // TODO: make this a "custom" IO specific to the CW305 target
+  // such that we can decouple this from the MIO signals.
```

Comment:
@vogelpi: does this trigger have to be a GPIO (i.e., are you writing to it in SW)?

If not, we could decouple this from the MIOs by adding it as a dedicated custom pad for the CW305 target (in the `add_pad` list in the hjson), and mapping it to the appropriate pin position using XDC constraints.

Reply:
It doesn't need to be a GPIO. But we want some software control over the trigger signal sent to the capture board. That's why we use the GPIO. So what we could do instead is:
- add a dedicated custom pad as you suggest, and
- drive this signal with AES idle signal AND the GPIO (to keep software control)
Do you think that would work?

Reply:
OK I see. That may work yes. Let me give this a look once we have merged this (when I align the current bronze pinout with what we had planned for silver).